common/src/main/java/com/microsoft/identity/common/internal/commands/parameters/GenerateShrCommandParameters.java

```diff
@@ -24,23 +24,27 @@
 
 import com.microsoft.identity.common.internal.authscheme.IPoPAuthenticationSchemeParams;
 
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.experimental.Accessors;
 import lombok.experimental.SuperBuilder;
 
 /**
  * Parameter class for generating SHRs.
  */
 @Getter
-@SuperBuilder
+@SuperBuilder(toBuilder = true)
+@EqualsAndHashCode(callSuper = true)
+@Accessors(prefix = "m")
 public class GenerateShrCommandParameters extends CommandParameters {
```

Comment:
FYI this class is also consumed by [OneAuth](https://office.visualstudio.com/OneAuth/_search?action=contents&text=GenerateShr&type=code&lp=code-Project&filters=ProjectFilters%7BOneAuth%7DRepositoryFilters%7BOneAuth%7D&pageSize=25&result=DefaultCollection%2FOneAuth%2FOneAuth%2FGBdev%2F%2Fmsal%2Fsource%2Fandroid%2Fmsal%2Fapp%2Fsrc%2Fmain%2Fjava%2Fcom%2Fmicrosoft%2Fidentity%2Finternal%2Fbroker%2FBrokerRequestConverter.java) -- I don't see anything here that looks breaking, but FYI in case you are

Reply:
This should be completely unnoticeable by OneAuth.